src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageServiceProtocol/LanguageServerTests.cs

```diff
@@ -1117,7 +1118,7 @@ public void TestPublishTokens(string text, int count, bool withAllowSideEffects,
         [InlineData("{\"A\": 1 }", "A+2", typeof(DecimalType))]
         [InlineData("{}", "\"hi\"", typeof(StringType))]
         [InlineData("{}", "", typeof(BlankType))]
-        [InlineData("{}", "{ A: 1 }", typeof(KnownRecordType))]
+        [InlineData("{}", "{ A: 1 }", typeof(UserDefinedRecordType))]
```

Comment:
>UserDefinedRecordType [](http://example.com/codeflow?start=45&length=21)

Confirming this change is intended?

Reply:
with this new change, it returned that type. not the knownrecordtype.